packages/manager/apps/pci-billing/src/components/consumption/useInstanceListColumns.tsx

```diff
@@ -87,11 +86,11 @@ export function useInstanceListColumns({
       id: 'action',
       cell: (row: TInstanceConsumptionDetail) => (
         <div className="min-w-[16rem]">
-          {['ok', 'resized'].includes(row.monthlyBilling?.status) && (
+          {['ok', 'resized'].includes(row.monthlyBilling?.status ?? '') && (
             <DataGridTextCell>
               {t('cpbc_hourly_instance_monthly_billing_since', {
                 since: useFormattedDate({
-                  dateString: row.monthlyBilling.since,
+                  dateString: row.monthlyBilling?.since ?? '',
```

Comment:
Maybe we could check first for monthlyBilling :
```typescript
{row.monthlyBilling && ['ok', 'resized'].includes(row.monthlyBilling.status ?? '') && (
```
and then remove the ?. for it, and even remove the `?? ''` if status and sicne are always defined